io.github.daviddesimone.opencloudsaves.yml

```diff
@@ -0,0 +1,68 @@
+app-id: io.github.daviddesimone.opencloudsaves
+runtime: org.gnome.Platform
+runtime-version: '42'
+sdk: org.gnome.Sdk
+command: bootstrap-opencloudsave.sh
+cleanup:
+  - /app/lib/sdk/golang
+finish-args:
+  - '--share=ipc'
+  - '--socket=fallback-x11'
+  - '--filesystem=host'
+  - '--socket=wayland'
+  - '--device=dri'
+  - '--share=network'
+build-options:
+  env:
+    - GOBIN=/app/bin
+    - GOROOT=/app/lib/sdk/golang
+modules:
+  - name: golang
```

Comment:
why this? There is a golang SDK

Reply:
I updated this to use the golang SDK extension. Earlier in my project I had a reason for doing it this way, but that reason is no longer relevant, and using the golang SDK extension works fine.